src/gui/sdlmain.cpp

```diff
@@ -243,21 +256,20 @@ struct SDL_Block {
 			bool display_res;
 		} full;
 		struct {
-			Bit16u width, height;
+			uint16_t width = 0; // TODO convert to int
+			uint16_t height = 0; // TODO convert to int
+			bool use_original_size = true;
 		} window;
 		Bit8u bpp;
-		Bit32u sdl2pixelFormat;
 		bool fullscreen;
-		bool lazy_fullscreen;
-		bool lazy_fullscreen_req;
-		bool vsync;
+		bool vsync = false;
```

Comment:
We might not even need vsync, because we're implicitly using double-buffering by using [SDL_RenderPresent](https://wiki.libsdl.org/SDL_RenderPresent), which describes writing to a back-buffer that's then flipped to and rendered.

Reply:
I still think we need vsync - e.g. for supporting adaptive sync in the future (it's the same setting basically, just a different value).

Reply:
Good stuff.  We can atleast shield the user from dealing with vsync; and manager this required sdl option internally until it proves useful to expose to the user.